coc/client.py

```diff
@@ -1520,6 +1572,25 @@ async def verify_player_token(self, player_tag: str, token: str) -> bool:
         data = await self.http.verify_player_token(player_tag, token)
         return data and data["status"] == "ok" or False
 
+    async def get_current_goldpass_season(self) -> GoldPassSeason:
+        """Get the current gold pass season
+
+        Raises
+        ------
+        Maintenance
+            The API is currently in maintenance.
+
+        GatewayError
+            The API hit an unexpected gateway exception.
+
+
+        Returns
+        --------
```

Comment:
```suggestion

        Returns
        --------
```
This needs consistent docstring too